sdcm/sct_config.py

```diff
@@ -1104,7 +1104,7 @@ class SCTConfiguration(dict):
              help="options will be used for enable encryption at-rest for tables"),
 
         dict(name="logs_transport", env="SCT_LOGS_TRANSPORT", type=str,
-             help="How to transport logs: rsyslog, ssh or docker", choices=("rsyslog", "ssh", "docker", "syslog-ng")),
+             help="How to transport logs: rsyslog, ssh or docker", choices=("rsyslog", "ssh", "docker", "syslog-ng", "kubectl")),
```

Comment:
I don't think we should update it. `kubectl` is used in parallel to one of the main log transports (syslogng or rsyslog).

Reply:
removed